app/models/disc_evidence_coversheet.rb

```diff
@@ -0,0 +1,75 @@
+class DiscEvidenceCoversheet
+  include ActiveModel::Model
+  include GovUk::DateAccessor
+
+  attr_accessor :claim_id
+  attr_writer :fee_scheme,
```

Comment:
why not add `:data_storage_type` to the call to `attr_accessor`?

Reply:
`super(@params)` handles setting it via activemode mixin. i have only added accessors and writers needed by the controller new action (`claim_id`), and readers are either defined individually here to specify defaults or additional processing required or, if no additional processing/defaults required then just a reader.

`data_storage_type` is the only attribute that does not require additional processing/defaults. 

i don't think it will break anything to make it an accessor so could simplify as you suggest but it is not required as creation/write is done via `super(@params)`